src/System Application/App/ControlAddIns/Resources/PowerBIManagement/js/PowerBIManagement.js

```diff
@@ -411,13 +430,6 @@ function CompileSettings() {
         settingsObject.background = models.BackgroundType.Transparent;
     }
 
-    if (_forceFitToPage) {
```

Comment:
I get that `forceFitToPage` option is removed, but it wasn't removed from the obsoleted procedure. So it's technically breaking (semantically) anyone using it without the grace period.

I'm ok with breaking if you are too :smile:, I'm just pointing it out.

Reply:
Let's respect the setting as long as it's still settable.